packages/itwinui-css/src/utils/icon/icon-component.scss

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@use 'sass:map';
+
+$icon-sizes: (
```

Comment:
Is there a benefit to adding this here and not using `$iui-icon-sizes` in `icon.scss` like were doing before?

Reply:
yes, updated description. it was resulting in too many changes